app/controllers/flow/property_entry_handler.rb

```diff
@@ -10,7 +10,7 @@ def model(session_data)
       end
 
       def form(params, session_data)
-        PropertyEntryForm.new(params.require(:property_entry_form).permit(*PropertyEntryForm::ENTRY_ATTRIBUTES)).tap do |model|
+        PropertyEntryForm.new(params.require(:property_entry_form).permit(*PropertyEntryForm::ENTRY_ATTRIBUTES, house_in_dispute: [])).tap do |model|
```

Comment:
I think this is a left-over from a different PR? Doesn't seem to be relevant here.